lib/Vaultaire/Daemon.hs

```diff
@@ -390,12 +392,15 @@ dayMapsFromCeph origin' = do
         Right day_map ->
             return $ Right (fromIntegral (BS.length contents), day_map)
 
+-- | Ceph object ID of the origin's Simple DayMap.
 simpleDayOID :: Origin -> ByteString
 simpleDayOID (Origin origin') = "02_" `BS.append` origin' `BS.append` "_simple_days"
```

Comment:
I would probably have used <> here if I had known of it's greatness back then. No need to change now.